pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -208,6 +216,11 @@ def __init__(self, name, **kw):
                            docstring=("Used for double weighted integration "
                                       "during qutrit readout"),
                            parameter_class=ManualParameter)
+        self.add_parameter('acq_weights_basis', vals=vals.Lists(),
+                           label="weight basis used",
+                           docstring=("Used to log the weights basis for integration "
+                                      "during qutrit readout"),
+                           parameter_class=ManualParameter)
```

Comment:
Basis in what space, this is left a bit confusing to me.

Reply:
eg. ['ge', 'gf'] or ['ge', 'ortho'] describing how weights were computed from the traces.
'ge' indicates (trace['g'] - trace['e']) and ortho refers to orthogonal to the previous (and spanning the 3 states)

Reply:
^ add this example to the docstring maybe, since the expected input is also not clear from the validator.

Reply:
I will fix myself.